### PR TITLE
Add top level disputes

### PR DESCRIPTION
**Pull request by kyleconroy**
r? @cupcait (not urgent, can wait til Monday) cc @michelle

### CODE REVIEW

File: lib/resources/Disputes.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+
+module.exports = StripeResource.extend({
+
+  path: 'disputes',
+
+  includeBasic: [
+    'list', 'retrieve', 'update', 'setMetadata', 'getMetadata'
```
Comment (cupcait) [inline lib/resources/Disputes.js]: What about set/get evidence methods?

Comment (kyleconroy) [inline lib/resources/Disputes.js]: Whoops, you're totally right, will need to add those.

Comment (kyleconroy) [inline lib/resources/Disputes.js]: So I've thought more about this, and I think a "setEvidence" method isn't needed, as the following works:

```js
stripe.disputes.update('dp_123', {evidence: {customer_name: 'Bob'}})
```

Comment (cupcait) [inline lib/resources/Disputes.js]: Yep, that sounds in line with how some of the other libraries do it. In that case, can you remove the metadata methods, too?

Comment (kyleconroy) [inline lib/resources/Disputes.js]: I think we should be consistent here: all other resources have the `setMetadata` and `getMetadata` methods, so a dispute object should be no different.

**Pull request merged**